tests/backup/backup_locked_bucket_test.go

```diff
@@ -406,3 +406,233 @@ var _ = Describe("{LockedBucketResizeOnRestoredVolume}", func() {
+				}
+			}
+		}
+		log.InfoD("Deploy applications")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				appNamespaces = append(appNamespaces, namespace)
+			}
+		}
+	})
+	It("Schedule backup while resizing the volume", func() {
+		providers := getProviders()
+		Step("Validate applications", func() {
+			ValidateApplications(contexts)
+		})
+		Step("Creating rules for backup", func() {
+			log.InfoD("Creating pre and post rule for deployed apps")
+			for i := 0; i < len(appList); i++ {
+				preRuleStatus, ruleName, err := Inst().Backup.CreateRuleForBackup(appList[i], orgID, "pre")
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating pre rule for deployed apps for %v with status %v", appList[i], preRuleStatus))
+				dash.VerifyFatal(preRuleStatus, true, "Verifying pre rule for backup")
+				preRuleNameList = append(preRuleNameList, ruleName)
+				postRuleStatus, ruleName, err := Inst().Backup.CreateRuleForBackup(appList[i], orgID, "post")
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating post rule for deployed apps for %v with status %v", appList[i], postRuleStatus))
+				dash.VerifyFatal(postRuleStatus, true, "Verifying post rule for backup")
+				postRuleNameList = append(postRuleNameList, ruleName)
+			}
+		})
+		Step("Creating cloud credentials", func() {
+			log.InfoD("Creating cloud credentials")
+			for _, provider := range providers {
+				credName = fmt.Sprintf("%s-%s-%v", "cred", provider, time.Now().Unix())
+				cloudCredUID = uuid.New()
+				cloudCredUIDMap[cloudCredUID] = credName
+				CreateCloudCredential(provider, credName, cloudCredUID, orgID)
+				log.InfoD("Created Cloud Credentials with name - %s", credName)
+			}
+		})
+		Step("Creating a locked bucket and backup location", func() {
+			log.InfoD("Creating locked buckets and backup location")
+			for _, provider := range providers {
+				for _, mode := range modes {
+					bucketName := fmt.Sprintf("%s-%s-%s", provider, getGlobalLockedBucketName(provider), strings.ToLower(mode))
+					backupLocation = fmt.Sprintf("%s-%s-%s-lock", provider, getGlobalLockedBucketName(provider), strings.ToLower(mode))
+					err := CreateS3Bucket(bucketName, true, 3, mode)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Creating locked s3 bucket %s", bucketName))
+					BackupLocationUID = uuid.New()
+					backupLocationMap[BackupLocationUID] = backupLocation
+					err = CreateBackupLocation(provider, backupLocation, BackupLocationUID, credName, cloudCredUID,
+						bucketName, orgID, "")
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Creating backup location %s", backupLocation))
+				}
+			}
+		})
+		Step("Register cluster for backup", func() {
```

Comment:
log.InfoD seems to be missing after the Step.

Reply:
How about Step("Creating source and destination cluster", func() { or something similar?